src/components/Editor/DraggableTabs/DraggableTab.js

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Draggable } from "@hello-pangea/dnd";
+import { useDispatch, useSelector } from "react-redux";
+import { Tab } from "react-tabs";
+import { setFocussedFileIndex } from "../EditorSlice";
+
+import './DraggableTabs.scss'
+
+const DraggableTab = ({children, panelIndex, fileIndex, ...otherProps}) => {
+  const openFiles = useSelector((state) => state.editor.openFiles)
+  const openFilesCount = openFiles[panelIndex].length
+  const dispatch = useDispatch()
+
+  const switchToFileTab = (panelIndex, fileIndex) => {
+    dispatch(setFocussedFileIndex({panelIndex, fileIndex}))
+  }
+
+  const onKeyPress = (e, panelIndex, fileIndex) => {
+    if (e.code === 'ArrowRight') {
+      switchToFileTab(panelIndex, (fileIndex + 1) % openFilesCount)
+    } else if (e.code === 'ArrowLeft') {
+      switchToFileTab(panelIndex, (fileIndex + openFilesCount - 1) % openFilesCount )
+    }
+  }
+  
+  return (
+    <Draggable  draggableId={`draggable${panelIndex}_${fileIndex}`} index={fileIndex}>
+      {(provided) => (
```

Comment:
you could use destructuring here to avoid prepending things with `provided.`, something like:

```
{({innerRef, draggableProps, dragHandleProps}) => (
```

I'm a bit rusty, but iirc you can also dip into draggableProps with `draggableProps: {style}` but I don't recall if you can do that in addition to grabbing the whole contents of the object?

you could also potentially omit the brackets in the example since it's the only param, but that might be a style thing so I'll leave it to you